tests/test_surface.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+import oddt
+from oddt.surface import (generate_surface_marching_cubes,
+                          find_surface_residues)
+from sklearn.utils.testing import (assert_array_equal,
+                                   assert_array_almost_equal,
+                                   assert_equal,
+                                   assert_almost_equal,
+                                   assert_in)
+import pandas as pd
+import numpy as np
+
+
+test_data_dir = os.path.dirname(os.path.abspath(__file__))
+protein = next(oddt.toolkit.readfile('pdb', os.path.join(
+    test_data_dir, 'data/pdbbind/10gs_protein.pdb')))
+protein.protein = True
+protein.addh(only_polar=True)
+
+def test_generate_surface_marching_cubes():
+    """Tests generating surfaces"""
+    verts1, faces1 = generate_surface_marching_cubes(protein, scaling=1., probe_radius=1.4)
+    verts2, faces2 = generate_surface_marching_cubes(protein, scaling=2., probe_radius=1.4)
+    verts3, faces3 = generate_surface_marching_cubes(protein, scaling=3., probe_radius=1.4)
+
+    # Higher scaling should result in a higher number of vertices
+    assert len(verts3) > len(verts2) > len(verts1), ('Higher scaling should result in '
+                                                     'a higher number of vertices')
+
+    ref_vert_shape_1 = (20178, 3)
+    ref_face_shape_1 = (40374, 3)
+    ref_vert_shape_2 = (75326, 3)
+    ref_face_shape_2 = (150700, 3)
+    ref_vert_shape_3 = (173634, 3)
+    ref_face_shape_3 = (347394, 3)
+
+    assert_equal(ref_vert_shape_1, verts1.shape)
+    assert_equal(ref_face_shape_1, faces1.shape)
+
+    assert_equal(ref_vert_shape_2, verts2.shape)
+    assert_equal(ref_face_shape_2, faces2.shape)
+
+    assert_equal(ref_vert_shape_3, verts3.shape)
+    assert_equal(ref_face_shape_3, faces3.shape)
+
```

Comment:
I think you should also check if input validation works fine, you can use `assert_raises` from `nose.tools`.